ElectronicObserver/Window/Tools/SortieRecordViewer/Sortie/Battle/BattleNormalDayFromNight.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Collections.Generic;
+using ElectronicObserver.KancolleApi.Types.ApiReqSortie.NightToDay;
+using ElectronicObserver.Window.Tools.SortieRecordViewer.Sortie.Battle.Phase;
+
+namespace ElectronicObserver.Window.Tools.SortieRecordViewer.Sortie.Battle;
+
+/// <summary>
+/// 通常/連合艦隊 vs 通常艦隊　夜昼戦 <br />
+/// api_req_sortie/night_to_day
+/// </summary>
+public sealed class BattleNormalDayFromNight : DayFromNightBattleData
+{
+	public override string Title => "対通常艦隊　夜昼戦";
```

Comment:
Missing translation (even if this is unused for now ?)

Reply:
fixed